.eslintrc.js

```diff
@@ -31,7 +31,11 @@ module.exports = {
 		'@typescript-eslint/no-dynamic-delete': 'off',
 		'@typescript-eslint/no-implied-eval': 'off',
 		'@typescript-eslint/strict-boolean-expressions': 'off',
+		'@typescript-eslint/no-unsafe-member-access': 'off',
+		'@typescript-eslint/no-unsafe-assignment': 'off',
+		'@typescript-eslint/no-unsafe-return': 'off',
```

Comment:
I suggest we should not return `any` from a function we define.

Reply:
yea I agree, but this check was checking more than that. 
Even with setup, it's still disallow `any`

Reply:
Can't understand even with setup. Can you share any example which was causing problem more than function return.

Reply:
seems like i was looking at the wrong error at that time, i cannot reproduce. Reverted the change =)